src/main/kotlin/com/hedvig/rapio/util/Errors.kt

```diff
@@ -23,3 +23,10 @@ fun unauthorized(message: String? = null) = HttpServerErrorException(
     message?.toByteArray(Charsets.UTF_8),
     Charsets.UTF_8
 )
+
+fun badRequest(message: String? = null) = HttpServerErrorException(
```

Comment:
Since we are throwing these, maybe we want to do it like this?
```suggestion
fun badRequestException(message: String? = null) = HttpServerErrorException(
```